UefiTestingPkg/AuditTests/PagingAudit/Windows/MemoryRangeObjects.py

```diff
@@ -135,10 +147,46 @@ def GetSystemMemoryType(self):
         if self.SystemMemoryType is None:
             return "None"
         try:
+            if (self.SystemMemoryType == 5 or self.SystemMemoryType == 6 or self.SystemMemoryType == 7):

```

Comment:
is SystemMemoryType really stored as an Index that needs agreement between C firmware and python?  Would rather see string so this less error prone to cross boundary.  

Also, adding a few comments in python code can make it much easier to read quickly.  The idea that a format string is embedded in the memory range and that is somehow tied to CpuNumber would be helpful.

Reply:
It is stored as a string in the MemoryInfoDatabase.dat file and I convert it to a list index to match the convention already present in the script. I can add some comments in the code.